### DISCUSSION TITLE
Membuat dashboard di shinydashboard

### DISCUSSION THREAD

**Discussion by sabrinaa311** [teamalgoritma/community#233] [Q&A] [answered]
kak, kalo mau membuat dashboard seperti ini, di fluidRow nya diisi dengan infoBox atau apa ya kak
![ShinyLEGO - Google Chrome 31_07_2022 16_17_08](https://user-images.githubusercontent.com/107026622/182019548-a876432c-c234-430d-b1ab-dcc9426ed06f.png)

**Comment by amaliasf** [comment, marked as answer]
Halo Bu @sabrinaa311 
Untuk membuat box berwarna biru tersebut Ibu bisa menggunakan function `box()` kemudian tambahkan parameter `background`.

Berikut referensi terkait:
- https://www.rdocumentation.org/packages/shinydashboard/versions/0.7.2/topics/box

Semoga membantu Bu :)

**Comment by sabrinaa311** [comment]
Kalo untuk menambahkan tulisan nya pake text input kah?

**Reply by amaliasf** [reply]
Untuk menambahkan tulisan bisa langsung dituliskan pada function `box()` tersebut:

Contoh:
```
      box(
        background = "light-blue",
        "Silahkan tuliskan teks disini"
      )
```

Namun apabila Ibu membutuhkan suatu text box yang dapat diberikan input, bisa menggunakan `textInput()`.

Semoga membantu Bu :)
